src/utilities/credentials.jl

```diff
@@ -25,11 +25,20 @@ end
 """
 Get the value for `key` in the `ini` file for a given `profile`.
 """
+_get_profile_value(ini::Inifile, profile::AbstractString, key::AbstractString; kwargs...) =
+    _get_ini_value(ini, "profile", profile, key; kwargs...)
+
+"""
+Get the value for `key` in the `ini` file for a given `sso_session`.
+"""
+_get_sso_session_value(ini::Inifile, sso_session::AbstractString, key::AbstractString; kwargs...) =
+    _get_ini_value(ini, "sso-session", sso_session, key; kwargs...)
+
 function _get_ini_value(
-    ini::Inifile, profile::AbstractString, key::AbstractString; default_value=nothing
+    ini::Inifile, type::AbstractString, type_value::AbstractString, key::AbstractString; default_value=nothing
 )
-    value = get(ini, "profile $profile", key)
-    value === :notfound && (value = get(ini, profile, key))
+    value = get(ini, "$type $type_value", key)
+    value === :notfound && (value = get(ini, type_value, key))
```

Comment:
This just seems weird now. We should probably just remove this and also define the following function:
```julia
function _get_ini_value(ini::Inifile, section_type::AbstractString, key::AbstractString; default_value=nothing)
```
However doing this is going to lead to ambiguous methods. I may need to rework this part of the code base.

Reply:
This should let us move forward for now.
```suggestion
    value === :notfound && (value = get(ini, section_name, key))
```